datashuttle/tui/screens/modal_dialogs.py

```diff
@@ -159,10 +169,16 @@ def __init__(self, mainwindow: App, path_: Optional[Path] = None) -> None:
         super(SelectDirectoryTreeScreen, self).__init__()
         self.mainwindow = mainwindow
 
+        self.available_drives = self.get_drives()
```

Comment:
because this variable is only used immediately under, and it is not necessary to track it's state, the `self` can be removed